Library/Homebrew/test/utils/tty_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "utils"
+
+describe Tty do
+  specify "::strip_ansi" do
```

Comment:
Can we use spec names that describe the behaviour rather than just naming a method where possible? These cases seem pretty straightforward, e.g. "::strip_ansi removes ANSI escapes from a String".